client/components/line-chart/index.js

```diff
@@ -86,18 +86,17 @@ class LineChart extends Component {
 
 	dateFormatFunction = ( displayMonthTicksOnly ) => ( date, index, tickRefs ) => {
 		const everyOtherTickOnly = ! displayMonthTicksOnly && tickRefs.length > X_AXIS_TICKS_MAX;
+		const matchingTicks = tickRefs
+			.map( ( tickRef, tickRefIndex ) =>
+				tickRef.__data__.getMonth() === date.getMonth() ? tickRefIndex : null
+			)
+			.filter( ( e ) => e !== null );
+		const meanTickRefs = matchingTicks.length
+			? matchingTicks.reduce( ( total, current ) => total + current, 0 ) / matchingTicks.length
+			: 0;
```

Comment:
This probably shouldn't default to `0`, as the mean of an empty set doesn't really have a defined value. Lodash defaults to `NaN` in this situation, which seems appropriate. I'm concerned that keeping it at `0` could cause some unexpected behaviour for `isFirstMonthTick`.

Reply:
Good catch! And agreed - the idea here is to preserve the existing behavior, so defaulting to`NaN` there makes the most sense.

Fixed in https://github.com/Automattic/wp-calypso/pull/46814/commits/e242fd4efcd9ba411131fa337e39789a118ab9c7 - feel free to take a look when you get a chance.